### DISCUSSION TITLE
Novu (notification center)

### DISCUSSION THREAD

**Discussion by rkcreation** [coollabsio/coolify#2483] [Service Template Requests]
https://docs.novu.co/self-hosting-novu/deploy-with-docker

**Comment by suplere** [comment]
Hello NOVU is great. I may share my current docker compose but now I wait for version 2 release (with framework support):

From DISCORD:
We’ve been working hard behind the scenes and have some exciting news about the next Novu Project release.

As you may have seen in Novu Cloud, we’ve released a completely new way to define and automate your notifications with the Novu Framework. This new code-based approach sets the foundation for the future of notifications and delivers greater power and complete flexibility to notification workflows.

Rest assured, these changes are coming to the Novu Project as well! We’re targeting an early October launch of an updated container image. Stay tuned for more details as we get closer!

In the meantime, thank you for your ongoing feedback and input. it has been critical in helping us shape Novu.

**Reply by christophemenager** [reply]
@suplere are you self hosting their v2 with coolify?

**Comment by anaclumos** [comment]
👍🏻 to this. Would love to see this happen.

**Comment by issam-seghir** [comment]
any guide how to self-host novu on coolify ?

**Comment by hunasdf** [comment]
+1

**Comment by mdroidhd** [comment]
+1
